### PR TITLE
Add data

### PR DESCRIPTION
**Pull request by BiancaMe**
Queries: 
1 -setting the species column to unspecified
image

2-commit a transition
image
image

3-delete inside a transition
image

4-savePoint in transition
image
image

Answers: 
How many animals are there?
image

How many animals have never tried to escape?

image

What is the average weight of animals?
image

Who escapes the most, neutered or not neutered animals?
image

What is the minimum and maximum weight of each type of animal?
image

What is the average number of escape attempts per animal type of those born between 1990 and 2000?

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -20,3 +20,34 @@ SELECT * FROM animals  WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
...
+DELETE FROM animals WHERE date_of_birth > '01/01/2022';
+SAVEPOINT delete_after;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO delete_after;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg  < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
```
Comment (shella12) [inline queries.sql]: - [x] Please include all queries from PR in to the `query.sql` file. You are currently missing following queries from `query.sql` file:

> 1. How many animals have never tried to escape?
> 2. What is the average weight of animals?
> 3. Who escapes the most, neutered or not neutered animals?
> 4. What is the minimum and maximum weight of each type of animal?
> 5. What is the average number of escape attempts per animal type of those born between 1990 and 2000? 

- [x] The query for question `What is the average number of escape attempts per animal type of those born between 1990 and 2000? ` will only showing results for `pokemon` species  becuse of the condition `where speicies= 'pokemon'`. Please include the result for other species born between 1990 and 2000 as well by removing the `where speicies= 'pokemon'` condition from the query.

File: data.sql [hunk 2]
```diff
@@ -7,3 +7,7 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Pikachu', '07-01-2021', 1, FALSE, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ( 'Devimon', '12-5-2020', 5, TRUE, 11);
+
+ALTER TABLE animals ADD species VARCHAR;
+
```
Comment (shella12) [inline data.sql]: - [x] Everything that creates or change the tables structure should be saved in the schema.sql file. Please move the `ALTER` statement form data.sql to schema.sql file.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @BiancaMe, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Added all queries ✅ 
- Used transactions ✅   
- Altered Animal table ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @BiancaMe  

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**